homework.md

```diff
@@ -44,37 +45,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is the Programming Language for the Web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Helps us to share information.
+Enables people to make their own changes to that information, and share it back.
+Allows us to make checkpoints so that we can save our work as we go along.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy the work and are given a link that allows you to work on the project without altering the original
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You are copying the repository so you can change it without touching the original
```

Comment:
also remember cloning is specific to copying onto your local device

Reply:
you should mention from remote to local